### DISCUSSION TITLE
Suggested Features

### DISCUSSION THREAD

**Discussion by emmettcrass** [keiwando/toonsquid#108] [Ideas]
I love Toonsquid, easily my favorite ipad animation app. Some features that would be cool to see would be:

1. Edit control points on all drawn vector lines.

2. Erase up to intersection of vector lines. (Helps make clean up in animation a ton faster)

3. Register colors in palettes so when a color in a palette is changed then it will change the color across all animation frames.

4. Ability to edit the Easing Curve Editor based on different keyframe attributes (Position, Scale, ect.) and more control in the editor.

5. Line tapering in the pen tool. 

6. Access to symbols from all projects across the app.

**Comment by keiwando** [comment]
Thanks for all of the feedback!

I'm not sure I fully understand no.4:
> Ability to edit the Easing Curve Editor based on different keyframe attributes (Position, Scale, ect.)

Could you maybe given an example of what exactly you mean by this?

**Reply by ArtistBirdy** [reply]
> Could you maybe given an example of what exactly you mean by this?

He means, this Easing Curve panel 👇

![image](https://user-images.githubusercontent.com/118920457/210603303-71be17b2-91ce-4f84-8628-c6a18ab08933.jpeg)

needs to be like this one👇

![image](https://user-images.githubusercontent.com/118920457/210603491-856743fa-ed12-4f14-8ef1-2c6db5f16b16.png)

for manually editing beziers
